command_line/show.py

```diff
@@ -250,7 +250,7 @@ def show_experiments(experiments, show_scan_varying=False):
     for i_expt, expt in enumerate(experiments):
         text.append("Experiment %i:" % i_expt)
         format_class = expt.imageset.get_format_class()
-        if format_class.__name__ != "Format":
+        if format_class.__name__ not in ["Format", "FormatMultiImage"]:
```

Comment:
```suggestion
        if not format_class.abstract():
```
Would this be more robust?

Reply:
Possibly, but if not:
```suggestion
        if format_class.__name__ not in {"Format", "FormatMultiImage"}:
```

Reply:
`abstract()` not always defined 🤔

and it turns out `FormatMultiImage` is not abstract, either

Reply:
Done subsequent suggestion instead.

Reply:
oh ffs it's `is_abstract`

Reply:
> oh ffs it's `is_abstract`

Probably yeh, will try that later